patchwork-events-entity/src/main/java/net/minecraftforge/event/entity/living/LivingDropsEvent.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Minecraft Forge, Patchwork Project
+ * Copyright (c) 2016-2020, 2019-2020
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation version 2.1
+ * of the License.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+
+package net.minecraftforge.event.entity.living;
+
+import java.util.Collection;
+
+import net.minecraftforge.common.MinecraftForge;
+
+import net.minecraft.entity.ItemEntity;
+import net.minecraft.entity.LivingEntity;
+import net.minecraft.entity.damage.DamageSource;
+
+/**
+ * LivingDropsEvent is fired when an Entity's death causes dropped items to appear.
+ *
+ * <p>This event is fired whenever an Entity dies and drops items in {@link LivingEntity#onDeath(DamageSource)}.</p>
+ *
+ * <p>
+ * {@link #source} contains the DamageSource that caused the drop to occur.<br>
+ * {@link #drops} contains the ArrayList of EntityItems that will be dropped.<br>
+ * {@link #lootingLevel} contains the amount of loot that will be dropped.<br>
+ * {@link #recentlyHit} determines whether the Entity doing the drop has recently been damaged.<br>
```

Comment:
Forge javadoc once again proves to be awful. A few minor nitpicks and a missed MCP name here:
```suggestion
 * {@link #source} contains the {@link DamageSource} that caused the drop to occur.<br>
 * {@link #drops} contains the Collection of {@link ItemEntity}s that will be dropped.<br>
 * {@link #lootingLevel} contains the level of Looting used to kill the entity doing the drop.<br>
 * {@link #recentlyHit} determines whether the entity doing the drop has recently been damaged by a player or tamed wolf.<br>
```